jsonpickle/util.py

```diff
@@ -43,10 +43,19 @@ def is_type(obj):
     >>> is_type(Klass)
     True
     """
+    # use "isinstance" and not "is" to allow for metaclasses
     if PY3:
-        return type(obj) is type
+        return isinstance(obj, type)
     else:
-        return type(obj) is type or type(obj) is types.ClassType
+        return isinstance(obj, (type, types.ClassType))
+
+
+def has_method(obj, name, *args, **kwargs):
+    try:
+        getattr(obj, name)(*args, **kwargs)
+        return True
+    except:
+        return False
```

Comment:
Oh yeah, sorry for missing your note -- yes, a tox recipe would be very cool. I've not used tox much myself (yet), but it sounds like I really should. I personally have a shell script that jumps into different virtualenvs and runs the JP unit tests using different pythons, but tox seems like a much nicer and official way to do that. :+1: